.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: python
 matrix:
   include:
+    - python: 3.8
+      dist: xenial
```

Comment:
Xenial is currently the default, but pinning it since it won't be the default forever.